### PR TITLE
update benchmark to have anti-affinity pods

### PR DESCRIPTION
**Pull request by tzneal**
Fixes #N/A 

**Description**

Updates the benchmark to have anti-affinity pods.

**How was this change tested?**

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -35,6 +35,8 @@ require (
 	sigs.k8s.io/controller-runtime v0.17.0
 )
 
+require github.com/evanphx/json-patch v4.12.0+incompatible // indirect
```
Comment (njtran) [inline go.mod:38]: Did you add this directly into the go.mod yourself? Curious why it's in a different block than the rest

Comment (tzneal) [reply go.mod:38]: go mod tidy added it. I imported sigs.k8s.io/controller-runtime/pkg/client/fake and its used there.

Comment (jonathan-innis) [reply go.mod:38]: Can we just manually shift it down? I think if you shift it into the lower block, `go mod tidy` should just respect where you put it IIRC.

**Review by jonathan-innis — APPROVED** [review]
/lgtm
/approve

**Pull request merged**